spec/features/bars/users_views_bar_details_spec.rb

```diff
@@ -1,7 +1,8 @@
 require "rails_helper"
 
 feature "non-user views a bar's details" do
-  let!(:bar1) { Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?") }
+  user1 = User.create(email: "a5@gmail.com", password: "12345678")
```

Comment:
For the next iteration, follow the pattern that you have in `user_edits_bar_spec.rb`

```
let(:new_user) { User.create(email: "a3@gmail.com", password: "12345678") }
let(:bar1) { Bar.create(name: "Happy Hour Lasagna", description: "Where are we at?", user: new_user) }
```